fedmsg_meta_umb/tests/test_tps.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# fedmsg_meta_umb is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# fedmsg_meta_umb is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with fedmsg; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# Authors:  Gowrishankar Rajaiyan <grajaiya@redhat.com>
+
+import fedmsg.tests.test_meta
+
+
+class TestTPSStart(fedmsg.tests.test_meta.Base):
+    """ TPS performs basic package sanity testing of rpms.
+
+    Messages (like the example given here) are published when an **TPS**
+    job is **starting**. (A TPS job performs basic sanity testing for the given brew build).
+    """
+    expected_title = 'tps.starting'
+    expected_packages = set(['systemd'])
```

Comment:
This is a good start!

I'd recommend getting `expected_subti` and `expected_link` working before we merge this.

`expected_icon` would be nice to have, but not strictly required.

Reply:
missing attributes and messages added.